src/Uno.UWP/Devices/Enumeration/Internal/Providers/Midi/MidiDeviceClassProviderBase.wasm.cs

```diff
@@ -0,0 +1,142 @@
+﻿#if __WASM__
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Uno.Devices.Midi.Internal;
+using Uno.Extensions;
+using Windows.Devices.Enumeration;
+using static Uno.Foundation.WebAssemblyRuntime;
+
+namespace Uno.Devices.Enumeration.Internal.Providers.Midi
+{
+	/// <summary>
+	/// Must be public to allow for WASM binding
```

Comment:
?

Reply:
Oh, obsolete comment 😀 . In the end, I extracted the static public dispatch method to a separate class
https://github.com/unoplatform/uno/blob/11ddc10953b603cef3d6f6eb58530685ed5fcb44/src/Uno.UWP/Devices/Enumeration/Internal/Providers/Midi/MidiDeviceConnectionWatcher.wasm.cs#L10